Changes

```diff
@@ -248,6 +248,7 @@ Working version
   (Gabriel Scherer, review by Nicolás Ojeda Bär and Florian Angeletti,
    report by Kate Deplaix)
 
+- #14190: bugfix;`ocaml -init init.ml -e "my code"` now works properly.
```

Comment:
There has been a `### Toplevel:` before (coming before "Manual and documentation"), but I'm not sure we need that here, given that it's an explicit flag which is changing:

```suggestion
- #14190: `ocaml -e` now also processes `-init` (previously it was ignored).
  (Emile Trotignon, review by David Allsopp and ???)
```